Os/Posix/Task.cpp

```diff
@@ -22,9 +23,17 @@ static const int SCHED_POLICY = SCHED_RR;
 
 typedef void* (*pthread_func_ptr)(void*);
 
+// Forward declaration
+int set_task_name(pthread_t thread, char* name);
```

Comment:
Minor: If these functions are used just in this file, I would declare them `static`